test/parallel/test-fs-read-position-validation.mjs

```diff
@@ -14,25 +14,21 @@ const length = buffer.byteLength;
 
 // allowedErrors is an array of acceptable internal errors
 // For example, on some platforms read syscall might return -EFBIG
-function testValidCbPositional([position, allowedErrors], callback) {
+function testValidCb([position, allowedErrors], callback) {
   fs.open(filepath, 'r', common.mustSucceed((fd) => {
-    fs.read(fd, buffer, offset, length, position, common.mustCall((err) => {
-      fs.close(fd, common.mustSucceed(callback));
+    let callCount = 3;
+    function handler(err) {
+      callCount--;
       if (err && !allowedErrors?.includes(err.code)) {
+        fs.close(fd, common.mustSucceed());
```

Comment:
Note that `mustSucceed` is not necessary here, because:
1. If you don't supply a callback to `fs.close`, by default it's throwing the exception.
1. We are throwing at the next line anyway.

But it doesn't hurt to have it, I just wanted to mention it.